packages/cli/create-strapi-app/templates/example/scripts/seed.js

```diff
@@ -248,3 +253,28 @@ async function importSeedData() {
   await importGlobal();
   await importAbout();
 }
+
+async function main() {
+  const { createStrapi, compileStrapi } = require('@strapi/strapi');
+
+  const appContext = await compileStrapi();
+  const app = await createStrapi(appContext).load();
+
+  app.log.level = 'error';
+
+  await seedExampleApp();
+  await app.destroy();
+
+  process.exit(0);
+}
+
+if (require.main === module) {
+  main().catch((error) => {
+    console.error(error);
+    process.exit(1);
+  });
+} else {
+  module.exports = {
+    seedExampleApp,
+  };
+}
```

Comment:
Do we use this pattern anywhere else? Might want to add a comment about it since it's a bit unusual.

Reply:
it's actually very common in node :) I'll remove it I just used it for testing